etc/greenboot/greenboot.conf

```diff
@@ -12,4 +12,11 @@ GREENBOOT_WATCHDOG_CHECK_ENABLED=true
 ### This variable is the number of hours after an upgrade that we consider
 ### the new deployment as culprit of reboot.
 ### It has to be a positive integer. Defaults to 24 (hours).
-# GREENBOOT_WATCHDOG_GRACE_PERIOD=24
\ No newline at end of file
+# GREENBOOT_WATCHDOG_GRACE_PERIOD=24
+
+### This variable allows you to specify healthchecks to be skipped. 
+### The healthcheck must be specified by script name, as in the 
+### example below. Multiple healthchecks may be skipped by separating
+### the script names with spaces.
+### DISABLED_HEALTHCHECKS=("01_repository_dns_check.sh 02_watchdog.sh")
```

Comment:
I would prefer some something simpler as user is going to modify them.

like: ``` ### DISABLED_HEALTHCHECKS=01_repository_dns_check.sh,02_watchdog.sh```

Reply:
Currently rewriting to use just an array, would that work? Currently looking something like this:

`DISABLED_HEALTHCHECKS=(script1_name script2_name)`

Is this more in line with what you have in mind?

Reply:
Nitpick, I think it would be better to double quote the script names for coherence with the other arrays we use in the code. 
ExtraMegaNitpick: I think is more clear to write the comment as a multiline, so instead of:
```bash
### DISABLED_HEALTHCHECKS=(01_repository_dns_check.sh 02_watchdog.sh)
```
Something like:
```bash
### DISABLED_HEALTHCHECKS=(
###     "01_repository_dns_check.sh"
###     "02_watchdog.sh"
### )
```

Reply:
I am starting to the like the format that @mmartinv suggested.
Before fully giving in to the idea, I would think form user perspective - Does this format makes it easy for some config preparer to easily update the configs.

Reply:
Updated to reflect the format suggested above, resolving this conversation